tests/fsharpqa/Source/Optimizations/GenericComparison/Equals09.il.bsl

```diff
@@ -1330,26 +1330,25 @@
               instance bool  Equals(object obj) cil managed
       {
         .custom instance void [mscorlib]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = ( 01 00 00 00 ) 
-        // Code size       22 (0x16)
+        // Code size       20 (0x14)
         .maxstack  4
         .locals init ([0] class Equals09/EqualsMicroPerfAndCodeGenerationTests/KeyWithInnerKeys V_0)
         .line 5,5 : 10,26 ''
         IL_0000:  ldarg.1
         IL_0001:  isinst     Equals09/EqualsMicroPerfAndCodeGenerationTests/KeyWithInnerKeys
         IL_0006:  stloc.0
         IL_0007:  ldloc.0
-        IL_0008:  brfalse.s  IL_0014
+        IL_0008:  brfalse.s  IL_0012
 
         .line 16707566,16707566 : 0,0 ''
         IL_000a:  ldarg.0
         IL_000b:  ldloc.0
-        IL_000c:  tail.
```

Comment:
I'm not totally comfortable with these tailcalls disappearing - have we discussed this? Thanks